theseus/src/launcher/download.rs

```diff
@@ -141,7 +141,7 @@ pub async fn download_assets(
     log::debug!("Loading assets");
     stream::iter(index.objects.iter())
         .map(Ok::<(&String, &Asset), crate::Error>)
-        .try_for_each_concurrent(Some(st.settings.read().await.max_concurrent_downloads), |(name, asset)| async move {
+        .try_for_each_concurrent(None, |(name, asset)| async move {
```

Comment:
Is the max_concurrent_downloads going to be used for anything else? If not, should probably keep the value used here or remove the setting entirely.

Reply:
It's not needed as the downloading limiting is handled by the semaphore (which is used inside of here). The semaphore is limited to max concurrent download's value, so only that many requests can be done at once. The reason it had problems earlier as this behavior wasn't working (it was fetching each file 3 times)